corehq/apps/data_cleaning/views/tables.py

```diff
@@ -147,12 +147,24 @@ def apply_all_changes(self, request, *args, **kwargs):
     @hq_hx_action("post")
     def undo_last_change(self, request, *args, **kwargs):
         self.session.undo_last_change()
-        return self.get(request, *args, **kwargs)
+        return self._trigger_clean_form_referesh(
```

Comment:
The commit message implies this is conditional, but the effect is that the form will always be refreshed with this `undo_last_change` and the `clear_all_changes` action, right?

Reply:
intention is not conditional... i think i'm saying we need to refresh it to clear that message in case it was shown